kore/test/Test/Kore.hs

```diff
@@ -83,10 +76,8 @@ import Kore.Internal.TermLike as TermLike hiding (
     Symbol,
  )
 import Kore.Log qualified as Log (
-    Entry (toEntry),
-    MonadLog (..),
     emptyLogger,
-    toEntry,
+    runLoggerT
```

Comment:
You have to run `scripts/fourmolu.sh` (in a `nix-shell` or using a local `fourmolu` installation) to check/adjust formatting (the RV repo runs the formatter automatically).
In this line, `fourmolu` wants a trailing comma.